types/generated/contentTypes.d.ts

```diff
@@ -666,6 +666,11 @@ export interface PluginUsersPermissionsUser extends Schema.CollectionType {
       'oneToMany',
       'api::comment.comment'
     >;
+    subscriptions: Attribute.Relation<
+      'plugin::users-permissions.user',
+      'oneToMany',
+      'api::subscription.subscription'
+    >;
     createdAt: Attribute.DateTime;
```

Comment:
@becevka is this file that you were talking about?
https://github.com/dev-launchers/strapiv4/pull/181#issuecomment-2106120482

Reply:
I think this is autogenerated content?